<launch>

  <!-- Arguments -->
  <arg name="image_dir" value="/home/bjoshi/colmap/test"/>
  <arg name="scale" value="1.0"/>
  <arg name="img_freq" value="30"/>
  <!-- <arg name="bag_file" value="/home/bjoshi/Downloads/cave_loop_w_cam_info.bag"/> -->

  <param name="/use_sim_time" value="true"/>

  <node name="republish1" type="republish" pkg="image_transport" output="screen" args="compressed in:=/gopro/image_raw raw out:=/cam0/image_raw" />
  
  <node name="bag_to_colmap" pkg="utils_ros" type="bag_to_colmap.py" output="screen"> 

    <remap from="/camera0" to="/cam0/image_raw"/> 

    <param name="image_dir" value="$(arg image_dir)"/>
    <param name="scale" value="$(arg scale)"/>
    <param name="img_freq" value="$(arg img_freq)"/>

  </node>

</launch>


